scipy/optimize/optimize.py

```diff
@@ -327,6 +327,8 @@ def fmin(func, x0, args=(), xtol=1e-4, ftol=1e-4, maxiter=None, maxfun=None,
         Set to True to print convergence messages.
     retall : bool, optional
         Set to True to return list of solutions at each iteration.
+    initial_simplex : array_like of shape (N + 1, N), optional
```

Comment:
It's not explicitly mentioned in this docstring what N is, although one can tell from the code. Is it worth giving a link to how to make a simplex?